src/shared/meson.build

```diff
@@ -435,6 +441,7 @@ libshared_deps = [threads,
                   libcap,
                   libcrypt,
                   libdl,
+                  libdw,
```

Comment:
Sorry, but this is not a good idea.

The code in libdw is extremely fragile, and we need to rework coredump to not try to do any parsing using libdw in the main process. In Fedora we are getting huge amounts of systemd-coredump crashes, and it's not only a huge annoyance, but it is most likely a security issue, since the coredump core is often running privileged. So week need to redo the design. I think pulling libdw into shared.so is just digging us deeper in the wrong direction.

Reply:
yeah I thought so as written above - safe_fork + dlopen?

Reply:
ok, I've moved the processing inside a safe_fork wrapper - I've pushed it as the first commit, so that it's easy to backport to v249-stable

Reply:
alright, switched to fork + dlopen